src/Serilog.Sinks.Splunk/Sinks/Splunk/SplunkJsonFormatter.cs

```diff
@@ -174,8 +182,11 @@ public void Format(LogEvent logEvent, TextWriter output)
                 JsonValueFormatter.WriteQuotedJsonString(logEvent.MessageTemplate.Text, output);
             }
 
-            output.Write(",\"RenderedMessage\":");
-            JsonValueFormatter.WriteQuotedJsonString(logEvent.RenderMessage(_formatProvider), output);
+            if (!_renderMessage)
```

Comment:
Should this logic be the other way round? ie `if (_renderMessage)` to match the _renderTemplate logic above?

Reply:
I think you are right. I removed the `!`

Reply:
thanks for the fix